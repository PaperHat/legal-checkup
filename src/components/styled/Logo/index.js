import styled from 'styled-components'

const Logo = styled.div`
  width: 96px;
  height: 27px;
  opacity: 0.39;
  font-family: Avenir;
  font-size: 32px;
  font-weight: 900;
  font-style: normal;
  font-stretch: normal;
  line-height: 0.84;
  letter-spacing: normal;
  color: #ffffff;
  display: inline-block;
  margin-top: 20px;
  margin-left: 10px;
  margin-bottom: 22px;
`
Logo.defaultName = 'StyledLogo'

export default Logo
