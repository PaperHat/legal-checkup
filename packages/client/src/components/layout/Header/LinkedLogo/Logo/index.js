import styled from 'styled-components'

const Logo = styled.div`
  background-color: #474747;
  border-radius: 5px;
  color: #ffffff;
  cursor: pointer;
  font-family: 'Museo Sans Cyrl 900', sans-serif;
  font-size: 10px;
  height: 35px;
  margin-bottom: 8px;
  margin-left: 5px;
  margin-top: -28px;
  object-fit: contain;
  opacity: 0.39;
  width: 32px;
`
Logo.displayName = 'Logo'

export default Logo
