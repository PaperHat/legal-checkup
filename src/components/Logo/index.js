import React from 'react'
import { StyledLogo } from '../styled'
import { Link } from 'react-router'

export default function Logo () {
  return (
    <StyledLogo>
      <Link to='./images/cl_logo.png' alt=''>
        Home
      </Link>
    </StyledLogo>
  )
}
