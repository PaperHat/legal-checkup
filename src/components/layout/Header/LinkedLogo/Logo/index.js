import styled from 'styled-components'
import styledMap from 'styled-map'

import { MOBILE, TABLET, DESKTOP } from '../../../../../constants'

const Logo = styled.div`
  cursor: ${({ active }) => (active ? 'default' : 'pointer')};

  ${styledMap('format', {
    [`${DESKTOP}`]: `
      height: 70px;
      width: 70px;
    `,
    [`${TABLET}`]: `
      height: 52px;
      width: 52px;
    `,
    [`${MOBILE}`]: `
      height: 32px;
      width: 32px;
    `
  })};
`
Logo.displayName = 'Logo'

export default Logo
