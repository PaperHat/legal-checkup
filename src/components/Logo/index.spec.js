import React from 'react'
import { shallow } from 'enzyme'

import Logo from '.'

describe('components:Logo', function () {
  it('renders the Logo and CSS properties properly', () =>
    expect(toJson(shallow(<Logo>Logo</Logo>))).toMatchSnapshot())
})
