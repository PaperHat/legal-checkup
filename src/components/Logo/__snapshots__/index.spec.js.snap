// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:Logo renders the Logo and CSS properties properly 1`] = `<StyledLogo />`;
